### PR TITLE
check frequency boost support on Arm

### PR DESCRIPTION
**Pull request by lubinsz**
Signed-off-by: Bin Lu <bin.lu@arm.com>

### CODE REVIEW

**Comment by lubinsz** [comment]
@balajismaniam
Hi,
Plz check it. Git rebase done.

File: source/pstate/pstate.go [hunk 1]
```diff
@@ -31,6 +32,12 @@ func (s Source) Name() string { return "pstate" }
 func (s Source) Discover() ([]string, error) {
 	features := []string{}
 
+	/* on Arm platform, the frequency boost mechanism is software-based */
```
Comment (marquiz) [inline source/pstate/pstate.go]: Small nitpick: I'd use '//' for the comment here

**Comment by lubinsz** [comment]
@marquiz 
Done.
Plz check it.
Thanks.

File: source/pstate/pstate.go [hunk 2]
```diff
@@ -31,6 +32,12 @@ func (s Source) Name() string { return "pstate" }
 func (s Source) Discover() ([]string, error) {
 	features := []string{}
 
+	//Skip pstate detection on Arm
```
Comment (marquiz) [inline source/pstate/pstate.go]: Another nitpick: space missing after double-slash.

**Comment by lubinsz** [comment]
@marquiz 
WOW~~~~
Thanks for your reminder, and sorry to bother you.

Please check it.
Thanks.

**Comment by lubinsz** [comment]
@marquiz 
Is it OK ?
Thanks.

**Comment by marquiz** [comment]
Looks good to me, now. Ping @balajismaniam

**Comment by marquiz** [comment]
I think we could merge this. @lubinsz: please rebase on latest master

**Comment by marquiz** [comment]
@lubinsz, please rebase and I'll merge this

**Comment by lubinsz** [comment]
@marquiz 
Rebase was done, please check it.
Thanks.

**Comment by marquiz** [comment]
Thanks! Merging was delayed because of my vacation

**Pull request merged**